src/views/Question.vue

```diff
@@ -4,12 +4,16 @@
       <div class="question-category">{{ question.title }}</div>
       <div class="question-text">{{ question.questionText }}</div>
         <div v-if="question.type === 'Single choice'">
-          <QuizSingleChoice :choices="question.answers"></QuizSingleChoice>
+          <QuizSingleChoice :choices="question.answers" @answering="select"></QuizSingleChoice>
         </div>
         <div v-else-if="question.type === 'Multiple choice'">
           <QuizMultiChoice :choices="question.answers"></QuizMultiChoice>
         </div> 
-      <button class='button-disabled'><v-icon name="arrow-right"/></button>
+
+        <button :class="buttonState" @click="submit" :disabled="inputSelected.length == 0">
```

Comment:
`buttonState` value here depends on the `answered` computed property which also checks for `inputSelected.length == 0`. Technically writing `:disabled="!answered"` should work the same way too.